unitex/src/fr/umlv/unitex/config/UserPreferences.java

```diff
@@ -152,7 +152,7 @@ public boolean setRecentDelas(String link) {
 			return false;
 		}
 		return true;
-	}                   
+	}               
```

Comment:
Make sure to remove trailing whitespace

Reply:
I'm not sure to understand, i removed the whitespace in this example.

Reply:
When I download UserPreferences.java, I see 15 spaces in the end of line 155. Also in lines 99 140 148...

Reply:
Right, I only removed tab.